config/environment.js

```diff
@@ -51,8 +49,7 @@ module.exports = function (environment) {
     ENV.APP.rootElement = "#ember-testing";
     ENV.APP.autoboot = false;
 
-    ENV.APP.AUTHENTICATION_URL = "/authenticate";
-    ENV.APP.DEVICE_URL = "/device";
+    ENV.APP.BASE_API_URL = "http://localhost:4200";
```

Comment:
```suggestion
    ENV.APP.BASE_API_URL = "";
```

We shouldn't tie this to a particular host if we don't have to. We'd also run via https locally. I'm not sure why we don't set it to `""` as the default anyway though and only override with `"https://ucentral.dpaas.arilia.com:16001"` for production? That would be the common way to do this in my experience.

Reply:
I'm setting the localhost url specifically because I'm using `URL` class to construct the urls which requires a valid `http` or `https` url, otherwise it throws an error. So `""` won't work, I don't see why this is a problem though, even if some test runner would use a different url or port, it's easy to parameterize.

Reply:
I see. Still not sure why we need a special value for the `test` env (I'd assume everything just works if you delete this line)? but feel free to merge.